src/lib/stores/pageContentStore.ts

```diff
@@ -0,0 +1,23 @@
+import { writable, type Writable } from 'svelte/store';
+
+export enum PageContent {
+  Dashboard = 'dashboard',
+  Login = 'login',
+  BecomeProvider = 'becomeProvider',
+  Reconnect = 'reconnect',
+}
+
+const createPageContentStore = () => {
+  const { subscribe, set } = writable(PageContent.Login);
+
+  return {
+    subscribe,
+    set,
+    login: () => set(PageContent.Login),
+    becomeProvider: () => set(PageContent.BecomeProvider),
+    dashboard: () => set(PageContent.Dashboard),
+    reconnect: () => set(PageContent.Reconnect),
```

Comment:
I don't think reconnect will be needed as we will have a modal that will be triggered onClick of a button. Assuming that's what this is being used for.

Reply:
We are not navigating to the modal. This will not be used so I think we need to delete it.